Gruntfile.js

```diff
@@ -36,6 +36,8 @@ const JS_FILES = [
 	`${SRC_DIR}/connectors/**/*.js`,
 	// Core files
 	`${SRC_DIR}/core/**/*.js`, `${SRC_DIR}/ui/**/*.js`,
+	// Shell Scripts
+	`${SRC_DIR}/shell-scripts/*.js`,
```

Comment:
You should use `scripts/*.js` instead of this, as scripts are in `scipts` directory, not in `src/shell-scripts`.

Reply:
Sorry for the multiple commits. You might want to squash this commit.

Reply:
I'll squash it anyway.